K8s Orion release branch (#2334)
RedisLabs/redislabs-docs: content/kubernetes/deployment/openshift/openshift-operatorhub.md

]
---

You can deploy Redis Enterprise for Kubernetes from the Red Hat OpenShift CLI. You can also use a UI, [OperatorHub](https://docs.openshift.com/container-platform/4.11/operators/index.html) (Red Hat) to install operators and create custom resources.

## Install the Redis Enterprise operator

1. Select **Operators > OperatorHub**.

2. Search for _Redis Enterprise_ in the search dialog and select the **Redis Enterprise Operator provided by Redis** marked as **Certified**.

  {{<note>}}
  Custom catalogs are not supported.
  {{</note>}}

3. On the **Install Operator** page, specify the namespace for the operator.

    Only one namespace per operator is supported.

4. Update the **channel** with the version you're installing.

>     For more information about specific versions, see the [release notes]({{<relref "/kubernetes/release-notes/">}}).

Reviewer: ```suggestion
    For more information about specific versions, see the [Kubernetes release notes]({{<relref "/kubernetes/release-notes/">}}).
```

5. Choose an approval strategy.

>     We recommend **Manual** for production systems to ensure the operator is only upgraded by approval.

Reviewer: ```suggestion
    Use **Manual** for production systems to ensure the operator is only upgraded by approval.
```

{{< note >}}
**Known Limitation** - The automatic use of the security constraint is limited. The

> Redis Enterprise must be named `rec` for the constraint to be used automatically.  **We recommend you use the cluster name `rec` when deploying with the OperatorHub.**

Reviewer: ```suggestion
Redis Enterprise must be named `rec` for the constraint to be used automatically.  **Use the cluster name `rec` when deploying with the OperatorHub.**
```
